.circleci/config.yml

```diff
@@ -145,7 +145,7 @@ commands:
             sudo apt-get update && sudo apt-get install -y subversion default-mysql-client
             sudo -E docker-php-ext-install mysqli
             mkdir -p /tmp/wordpress
-            ./vendor/bin/wp core download --path=/tmp/wordpress
+            ./vendor/bin/wp core download --path=/tmp/wordpress --version=nightly
```

Comment:
We should ensure this doesn't make it into master, although it should stay for the moment for testing purposes.

Reply:
Yes. Good point. I'll keep your comment as a reminder to remove this line before merging ;)

Reply:
This also applies to https://github.com/godaddy-wordpress/coblocks/pull/1923/files#diff-9fd21b09a0144355cfd001e3952767df08b2e04241994470aa1d627f797b171e unless we want wp-env to stay on nightly builds.

Reply:
Yeah, for the .wp-env.json config file it's a little bit different, as it's only for the local execution. Maybe it shouldn't be there, but it matters less if it's merged or not, because it doesn't impact the CI.

Reply:
Are we ready to revert these changes @olafleur @AnthonyLedesma ?

Reply:
Yes. I will get these reverted.